### DISCUSSION TITLE
Korean Language Leader

### DISCUSSION THREAD

**Discussion by limitless00net** [OHF-Voice/intents#1397] [Language Leaders]
Following #1350 

As current Korean reviewer in Lokalise, I want to apply to be a Korean language leader.

I submitted 3 PRs #1394 #1395 #1396 today.

**Comment by synesthesiam** [comment]
Thanks @limitless00net! I'm adding you as a Korean language leader now. I've done my best to merge the work done by you and @miumida. Please take a look and see what you think :slightly_smiling_face:

**Reply by limitless00net** [reply]
Thank you for adding me as a Korean language leader. I appreciate the opportunity.
